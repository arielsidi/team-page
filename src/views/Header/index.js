import React from "react";
import styled from "styled-components";

import logoImage from "../../assets/woffu-logo.png";

const StyledHeader = styled.header`
    background: ${props => props.theme.primaryColor};
    width: 100%;
    display: flex;
    justify-content: center;
    align-items: center;
    padding: ${props => props.theme.panelSpacing}px;
`;

const Logo = styled.h1`
    margin: 0;
    padding-top: 5px;
    > img {
        width: 91px;
        height: 25px;
    }
    > span {
        color: #ffffff;
        margin-left: 4px;
        font-size: ${props => props.theme.fontSizeXsmall};
        font-weight: ${props => props.theme.fontWeightLight};
        opacity: 0.8;
    }
`;

const Header = () => (
    <StyledHeader>
        <Logo>
            <img src={logoImage} alt="Woffu HR Suite" />
            <span>HR Suite</span>
        </Logo>
    </StyledHeader>
);

export default Header;
