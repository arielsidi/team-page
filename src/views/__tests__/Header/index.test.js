import React from "react";
import Header from "../../Header";
import renderer from "react-test-renderer";

describe("Header", () => {
    it("Should render properly.", () => {
        const tree = renderer.create(<Header />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
