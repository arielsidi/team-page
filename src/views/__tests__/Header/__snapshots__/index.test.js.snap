// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render properly. 1`] = `
<header
  className="Header__StyledHeader-sc-1xllh7p-0 cMByFa"
>
  <h1
    className="Header__Logo-sc-1xllh7p-1 rahGx"
  >
    <img
      alt="Woffu HR Suite"
      src=""
    />
    <span>
      HR Suite
    </span>
  </h1>
</header>
`;
